tools/CouchDB_Import_Demographics.php

```diff
@@ -135,7 +167,7 @@ function _generateQuery() {
             $EDCFields = ", c.EDC as EDC";
             $fieldsInQuery .= $EDCFields;
         }
-        $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND c.Entity_type != 'Scanner'";
+        $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND c.Entity_type != 'Scanner'"." GROUP BY s.ID";
```

Comment:
Syntax error or access violation: 1055 Expression #14 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'loris.pc_comment.Value' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by